src/algebra/symmetrized.lean

```diff
@@ -210,4 +210,32 @@ lemma mul_comm [has_mul α] [add_comm_semigroup α] [has_one α] [invertible (2
   a * b = b * a :=
 by rw [mul_def, mul_def, add_comm]
 
+instance [ring α] [invertible (2 : α)] : is_comm_jordan αˢʸᵐ :=
+{ mul_comm := sym_alg.mul_comm,
+  lmul_comm_rmul_rmul := λ a b, begin
+    -- Rearrange LHS
+    --(commute.one_left (unsym (a*a))).bit0_left.inv_of_left
```

Comment:
Maybe inline it instead as
```suggestion
   have commute_two_left := \lam a : α, (commute.one_left a).bit0_left,
```
and inline that below.

Reply:
I've added `have commute_half_left ...`